week3/project/ecommerce/src/hooks/useFavoriteItems.jsx

```diff
@@ -0,0 +1,28 @@
+import { useContext } from 'react'
+import { FavoriteIdItemContext } from '../contexts/FavoriteIdItemContext';
+import svgReg from '../images/heart-regular.svg'
+import svgSol from '../images/heart-solid.svg';
+
+export default function useFavoriteItems(item) {
```

Comment:
I see that when you used this hook you give it item id as argument so it's better to rename the parameter

✏️need to modify✏️